# Translations template for neutron.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the neutron project.
#
# Translators:
msgid ""
msgstr ""
"Project-Id-Version: Neutron\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-06-20 06:08+0000\n"
"PO-Revision-Date: 2014-05-29 07:49+0000\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: French (http://www.transifex.com/projects/p/neutron/language/"
"fr/)\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: neutron/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Exception d'origine en cours de suppression : %s"

#: neutron/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Exception inattendue survenue %d fois... Nouvel essai."

#: neutron/openstack/common/periodic_task.py:179
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr ""

#: neutron/openstack/common/service.py:190
msgid "Exception during rpc cleanup."
msgstr "Exception pendant le nettoyage rpc."

#: neutron/openstack/common/service.py:279
msgid "Unhandled exception"
msgstr "Exception non gérée"

#: neutron/openstack/common/db/api.py:72
msgid "DB exceeded retry limit."
msgstr "DB dépassé limite de tentatives"

#: neutron/openstack/common/db/api.py:76
msgid "DB connection error."
msgstr "erreurs de connexion DB"

#: neutron/openstack/common/db/sqlalchemy/session.py:460
msgid "DB exception wrapped."
msgstr "Exception BD encapsulée."

#: neutron/openstack/common/middleware/notifier.py:40
#, python-format
msgid "An exception occurred processing the API call: %s "
msgstr ""

#: neutron/openstack/common/rpc/amqp.py:480
#: neutron/openstack/common/rpc/impl_zmq.py:286
msgid "Exception during message handling"
msgstr "Exception pendant le traitement des messages"

#: neutron/openstack/common/rpc/common.py:88
msgid "Exception in string format operation"
msgstr "Exception dans l'opération de format de chaîne"

#: neutron/openstack/common/rpc/common.py:292
#, python-format
msgid "Returning exception %s to caller"
msgstr "Renvoi de l'exception %s à l'appelant"

#: neutron/openstack/common/rpc/impl_kombu.py:156
msgid "Failed to process message ... skipping it."
msgstr "Echec de traitement du message... Message ignoré."

#: neutron/openstack/common/rpc/impl_kombu.py:160
msgid "Failed to process message ... will requeue."
msgstr "Échec de traitement du message... Message remis en file d'attente."

#: neutron/openstack/common/rpc/impl_kombu.py:571
#, python-format
msgid ""
"AMQP server on %(hostname)s:%(port)d is unreachable: %(err_str)s. Trying "
"again in %(sleep_time)d seconds."
msgstr ""
"Le serveur AMQP sur %(hostname)s:%(port)d est inaccessible : %(err_str)s. "
"Nouvelle tentative dans %(sleep_time)d secondes."

#: neutron/openstack/common/rpc/impl_kombu.py:625
#: neutron/openstack/common/rpc/impl_qpid.py:575
#, python-format
msgid "Failed to declare consumer for topic '%(topic)s': %(err_str)s"
msgstr ""
"Echec de la déclaration du consommateur pour la rubrique '%(topic)s' : "
"%(err_str)s"

#: neutron/openstack/common/rpc/impl_kombu.py:647
#: neutron/openstack/common/rpc/impl_qpid.py:594
#, python-format
msgid "Failed to consume message from queue: %s"
msgstr "Echec de la consommation du message depuis la file d'attente : %s"

#: neutron/openstack/common/rpc/impl_kombu.py:686
#: neutron/openstack/common/rpc/impl_qpid.py:629
#, python-format
msgid "Failed to publish message to topic '%(topic)s': %(err_str)s"
msgstr ""
"Echec de la publication du message dans la rubrique '%(topic)s' : %(err_str)s"

#: neutron/openstack/common/rpc/impl_qpid.py:191
msgid "Failed to process message... skipping it."
msgstr "Echec du traitement du message... Message ignoré."

#: neutron/openstack/common/rpc/impl_qpid.py:517
#, python-format
msgid "Unable to connect to AMQP server: %(e)s. Sleeping %(delay)s seconds"
msgstr ""
"Impossible de se connecter au serveur AMQP : %(e)s. En veille pendant "
"%(delay)s secondes"

#: neutron/openstack/common/rpc/impl_qpid.py:602
msgid "Error processing message.  Skipping it."
msgstr "Erreur lors du traitement du message. Message ignoré."

#: neutron/openstack/common/rpc/impl_zmq.py:96
msgid "JSON serialization failed."
msgstr "Echec de la sérialisation JSON."

#: neutron/openstack/common/rpc/impl_zmq.py:195
msgid "ZeroMQ socket could not be closed."
msgstr ""

#: neutron/openstack/common/rpc/impl_zmq.py:339
msgid "RPC message did not include method."
msgstr "Le message d'appel de procédure distante n'a pas inclus la méthode."

#: neutron/openstack/common/rpc/impl_zmq.py:476
msgid "Topic socket file creation failed."
msgstr "La création du fichier socket de la rubrique a échoué."

#: neutron/openstack/common/rpc/impl_zmq.py:482
#, python-format
msgid ""
"Local per-topic backlog buffer full for topic %(topic)s. Dropping message."
msgstr ""
"Mémoire tampon de commandes en attente par rubrique locale saturée pour la "
"rubrique %(topic)s. Suppression du message."

#: neutron/openstack/common/rpc/impl_zmq.py:498
#, python-format
msgid "Required IPC directory does not exist at %s"
msgstr "Répertoire IPC requis n'existe pas à %s"

#: neutron/openstack/common/rpc/impl_zmq.py:507
#, python-format
msgid "Permission denied to IPC directory at %s"
msgstr "Permission refusée au répertoire IPC à %s"

#: neutron/openstack/common/rpc/impl_zmq.py:510
msgid "Could not create ZeroMQ receiver daemon. Socket may already be in use."
msgstr ""
"Impossible de créer le démon récepteur ZeroMQ. Le socket est sans doute déjà "
"en cours d'utilisation."

#: neutron/openstack/common/rpc/impl_zmq.py:563
msgid "ZMQ Envelope version unsupported or unknown."
msgstr "Version de l'enveloppe ZMQ non prise en charge ou inconnue."
